src/NUnitFramework/tests/TestContextTests.cs

```diff
@@ -400,7 +400,16 @@ public void FilePathAndDescriptionDoesNotThrow()
 #endif
         public void InvalidFilePathsThrowsArgumentException(string filePath)
         {
+#if NETCOREAPP2_0
```

Comment:
Good find!

This test is only intended to test that a garbage path is checked for, and thrown out. The test below checks the behaviour for non-existent files already.

I think a better change here would be to make the test path something which is invalid on .NET Core too - i.e. change it to `bad|path.png`?